common/lib/xmodule/xmodule/css/video/display.scss

```diff
@@ -74,70 +74,68 @@
     width: 0px;
     height: 0px;
   }
-
-  .wrapper-downloads {
-    margin: 0;
-    padding: 0;
-
-    .video-download-button {
-      display: inline-block;
-      vertical-align: top;
-      margin: ($baseline*0.75) ($baseline/2) 0 0;
-
-      > a {
-        @include transition(all $tmg-f2 ease-in-out 0s);
-        @include font-size(14);
-        line-height : 14px;
-        float: left;
-        border-radius: 3px;
-        background-color: $very-light-text;
-        padding: ($baseline*0.75);
-        color: $lighter-base-font-color;
-
-        &:hover,
-        &:focus {
-          background-color: $action-primary-active-bg;
-          color: $very-light-text;
-        }
-      }
-    }
-
-    .video-tracks {
-
-      > a {
-        border-radius: 3px 0 0 3px;
-      }
-
-      > a.external-track {
-        border-radius: 3px;
-      }
+  
+    .downloads-heading {
+        margin: 1em 0 0 0;
     }
+  
+    .wrapper-downloads {
+        display: flex;
+        
+        .hd {
+            margin: 0;
+        }
+        
+        .wrapper-download-video,
+        .wrapper-download-transcripts,
+        .wrapper-handouts,
+        .branding {
+            flex: 1;
+            margin-top: $baseline;
+            @include padding-right($baseline);
+            vertical-align: top;
+        }
+        
+        .wrapper-download-video {
+            
+            .video-sources {
+                margin: 0;
+            }
+        }
+        
+        .wrapper-download-transcripts {
+            
+            .list-download-transcripts {
+                margin: 0;
+                padding: 0;
+                list-style: none;
+                
+                .transcript-option {
+                    margin: 0;
+                }
+            }
+        }
+        
+        .branding {
+            @include padding-right(0);
 
-    .branding {
-      display: inline-block;
-      float: right;
-      margin: 15px 0 0 10px;
-      vertical-align: top;
-
-      .host-tag {
-        @include margin-right($baseline/2);
-        position: absolute;
-        left: -9999em;
-        display: inline-block;
-        vertical-align: middle;
-        font-size: 70%;
-        color: #777;
-      }
+            .host-tag {
+                position: absolute;
+                left: -9999em;
```

Comment:
No need for RTL here as it's meant to be off-screen regardless. Also, only relevant for XuetangX.